client/lib/abtest/active-tests.js

```diff
@@ -77,4 +77,12 @@ module.exports = {
 		},
 		defaultVariation: 'add'
 	},
+	headstart: {
+		datestamp: '20160203',
+		variations: {
+			original: 20,
+			headstart: 80
```

Comment:
Technically, "original" and "notTested" would behave the exact same - we'd just ignore "notTested" in our results.